MIGRATION.md

```diff
@@ -5,6 +5,8 @@
 
 o-tracking now uses the Beacon API in all browsers which support it. In v2 this was an opt-in feature, now it is always enabled in browsers which support it. For browsers which do not support Beacon API o-tracking will revert to the XMLHTTPRequest API.
 
+o-tracking now uses the Crypto, TextEncoder, and Uint8Array APIs, to support older browser please include polyfills for these features.
```

Comment:
Does polyfill.io cover all of these? A link or some more instruction on how would be helpful for people migrating :) especially for Build Service users, I think it's common that o-tracking is included with the Build Service -- might even be worth flagging in the readme if needed for IE11?

Reply:
polyfill.io does not have Crypto.subtle but does have the rest of them

Reply:
I'm running the tests on browserstack across all our supported browsers to confirm it works